pkgs/by-name/hy/hyperspeedcube/package.nix

```diff
@@ -0,0 +1,131 @@
+{ cmake
+, alsa-lib
+, atk
+, cairo
+, directx-shader-compiler
+, expat
+, fetchFromGitHub
+, fontconfig
+, freetype
+, gdk-pixbuf
+, glib
+, gtk3
+, harfbuzz
+, lib
+, libGL
+, libX11
+, libxcb
+, libXcursor
+, libXi
+, libxkbcommon
+, libXrandr
+, makeWrapper
+, mold
+, pango
+, pkg-config
+, python3
+, rustPlatform
+, shaderc
+, stdenv
+, vulkan-extension-layer
+, vulkan-headers
+, vulkan-loader
+, vulkan-tools
+, vulkan-tools-lunarg
+, vulkan-validation-layers
+, wayland
+, wrapGAppsHook
```

Comment:
`wrapGAppsHook` should be either `wrapGAppsHook3` for gtk3 apps or `wrapGAppsHook4` for gtk4 apps.